src/components/DateMonthPicker/index.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from 'react'
+import { Button, translate, Icon, Bold } from 'cozy-ui/transpiled/react'
+import range from 'lodash/range'
+import { format } from 'date-fns'
+
+const gridStyle = {
+  display: 'grid',
+  gridTemplateColumns: 'repeat(4, auto)',
+  gridTemplateRows: 'repeat(3, 3fr)',
+  overflow: 'hidden'
+}
+
+const yearButtonStyle = {
+  flexGrow: 0,
+  minWidth: '3rem',
+  minHeight: '3rem',
+  textAlign: 'center',
+  borderWidth: 0,
+  margin: 0,
+  color: 'var(--coolGrey)',
+  background: 'none !important'
+}
+
+const monthButtonStyle = {
+  minHeight: '3rem',
+  minWidth: 0,
+  margin: 0,
+  border: 0,
+  borderRadius: 0,
+  fontWeight: 'normal',
+  textTransform: 'none'
+}
+
+const selectedStyle = {
+  color: 'var(--primaryColor)',
+  fontWeight: 'bold'
+}
+
+const MonthButton = translate()(({ monthNum, f, onClick, isSelected }) => {
+  const d = new Date(2019, monthNum, 15)
```

Comment:
Looks like the year and day should not be static here.

Reply:
Fixed in UI, I added a comment explaining why year and day are not important here.